examples/hacker_news_assets/README.md

```diff
@@ -1,109 +1,46 @@
-# Hacker News Demo Jobs - Software-Defined Assets Version
+# Hacker News Demo
 
-This repo used Dagster's experimental software-defined asset APIs to set up a "realistic" example of using dagster in a production world, integrating with
-many of its features, including:
+This is meant to be a full "realistic" demo of Dagster, that takes advantage of many of its features, including:
 
-- Schedules / Sensors
-- Asset Materializations
+- Software-defined assets
+- Schedules
+- Sensors
 - IOManagers
 - Resources
-- Unit Tests
-- dbt, snowflake, s3, and pyspark integrations
+- dbt, Snowflake, S3, and PySpark integrations
```

Comment:
the example still uses a snowflake io manager that's written in `resources/snowflake_io_manager.py` not our built in io manager

Reply:
good point